### PR TITLE
fixes CC-867: comment out /etc/exports entries that conflict with serviced exports

### PR DESCRIPTION
**Pull request by wplu**
https://jira.zenoss.com/browse/CC-867

DEMO1 - unit test passes:
```
~/src/europa/src/golang/src/github.com/control-center/serviced/dfs/nfs
# plu@plu-9: go test -v -run TestWriteExports
=== RUN TestWriteExports
--- PASS: TestWriteExports (0.00s)
	server_test.go:163: created temp dir: /tmp/nfs_unit_tests_464387770
PASS
ok  	github.com/control-center/serviced/dfs/nfs	0.009s
```

DEMO2 - comment out /etc/exports entries that conflict with serviced exports:
```
# plu@plu-9: cat /etc/exports 
/exports	*(rw,fsid=0,no_root_squash,insecure,no_subtree_check,async)
/exports/serviced_var_volumes	*(rw,no_root_squash,nohide,insecure,no_subtree_check,async)

/space *(ro)

# plu@plu-9: zendev serviced -dx
...

# plu@plu-9: cat /etc/exports 
# serviced removed: /exports	*(rw,fsid=0,no_root_squash,insecure,no_subtree_check,async)
# serviced removed: /exports/serviced_var_volumes	*(rw,no_root_squash,nohide,insecure,no_subtree_check,async)

/space *(ro)


# --- SERVICED EXPORTS BEGIN ---
# --- Do not edit this section
/exports	*(rw,fsid=0,no_root_squash,insecure,no_subtree_check,async)
/exports/serviced_var_volumes	*(rw,no_root_squash,nohide,insecure,no_subtree_check,async)
# --- SERVICED EXPORTS END ---

```

### CODE REVIEW

**Comment by jafcrocker** [comment]
Given this change, should we even bother with the start/end markers anymore?  It would work without them, no?

File: dfs/nfs/server.go [hunk 1]
```diff
@@ -262,12 +262,32 @@ func (c *Server) writeExports() error {
...
+	filteredContent := ""
+	scanner := bufio.NewScanner(strings.NewReader(originalContents))
+	for scanner.Scan() {
+		line := strings.TrimSpace(scanner.Text())
+		if !strings.HasPrefix(line, "#") {
+			fields := strings.Fields(line)
+			if len(fields) > 0 {
+				mountpoint := fields[0]
+				if _, ok := mountpaths[mountpoint]; ok {
+					continue
```
Comment (jafcrocker) [inline dfs/nfs/server.go]: Perhaps instead of removing the line, we should comment it out.

Comment (wplu) [inline dfs/nfs/server.go]: yes, that is far superior.  implementing now

**Comment by wplu** [comment]
yes, it would work without begin/end markers, but it helps to alert the sysadmin who modifies this file

**Pull request merged**